src/dialogs/more-info/controls/more-info-vacuum.ts

```diff
@@ -68,7 +68,7 @@ const VACUUM_COMMANDS: VacuumCommand[] = [
   },
   {
     translationKey: "clean_spot",
-    icon: "hass:broom",
+    icon: "hass:target",
```

Comment:
Maybe `target-variant`? As spots cleaning is mostly square?
```suggestion
    icon: "hass:target-variant",
```